srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -13,6 +18,15 @@ def render_GET(self, request: Request):
         content = f"I am request #{self.numberRequests}\n"
         return content.encode("ascii")
 
+    def render_HEAD(self, request: Request):
+        request.setResponseCode(HTTPStatus.OK)
+        request.setHeader(b"Server", f"{PRODSTRING}")
+        request.setHeader(b"Access-Control-Allow-Origin", "*")
+        request.setHeader(b"content-type", b"text/plain")
+        content = f"I am HEAD request #{self.numberRequests}\n"
+        logger.warning("Request type is %s", request.method)
```

Comment:
Update log level to be info and and try to match what is currently being logged.